content/rule_book/roles_and_responsibilities.md

```diff
@@ -0,0 +1,97 @@
+# Roles and Responsibilities
+
+## Team Organization
+
+A team can consist of up to 5 quizzers. Teams can have up to 4 quizzers seated in a quiz at any given time.
```

Comment:
**Alan Yam:** In section 2.1 (team organization), the maximum number of quizzers on a team is mentioned, but there is no mention of a minimum.

Reply:
The rule book hasn't defined a minimum number of quizzers on a team in past editions of the rule book. In order to maintain FE we did not mention minimums when looking at the size of an individual team. 

Personally at the district level I have seen teams of 2 out compete teams of 5 just because they had better content and knew their stuff.

Reply:
Agreed Zach, but I don't even feel the need to discuss "ability to compete" here. The key point is that the current rule book has zero mention of team size minimums. If something was clearly stated in the current rule book (or clearly NOT stated in this case), we weren't changing that. 

If those team size minimums are desired, that would be a version 2 suggestion. 

(I know of at least WCD, which has a local rule for team size minimums, which I believe is more logistics-driven than competitive-driven, but I don't know that for sure. And of course, districts have complete freedom to set their own local rules).

Reply:
Agreed. That was poor wording on my part. I was simply adding that as an example to help show that I don't think a minimum needs to be set since in past experience I haven't seen much of a difference between teams of 5 and smaller teams. 

But again that discussion would need to occur only if it were proposed as a rule change in upcoming versions, not for this version.

Thank you for clarifying.